eslintrc.json

```diff
@@ -558,29 +559,15 @@
     "**/ui/public/**/*.js"
   ],
   "overrides": [
-    {
-      "files": "*.vue",
-      "globals": {
-        "apos": true,
-        "defineProps": "readable",
-        "defineEmits": "readable",
-        "defineExpose": "readable",
-        "defineOptions": "readable",
-        "defineModel": "readable",
-        "defineSlots": "readable"
-      }
-    },
```

Comment:
same config for js frontend and vue files. These globals available in vue plugin.